reconcile/dashdotdb_slo.py

```diff
@@ -82,12 +83,21 @@ def _post(self, service_slos: Iterable[ServiceSLO]) -> Response | None:
             if self.dry_run:
                 continue
 
+            if isnan(item.value):
+                LOG.info(
+                    f"{self.logmarker} Skipping SLO '{slo_name}' in SLO doc '{item.slo_doc_name}'"
+                    "as the obtained value is not a number"
+                )
+                continue
+
             LOG.info("%s syncing slo %s", self.logmarker, slo_name)
-            response = self._do_post(endpoint, payload)
             try:
+                response = self._do_post(endpoint, payload)
                 response.raise_for_status()
             except requests.exceptions.HTTPError as details:
                 LOG.error("%s error posting %s - %s", self.logmarker, slo_name, details)
+            except requests.exceptions.InvalidJSONError as details:
+                LOG.error("%s error posting %s - %s", self.logmarker, slo_name, details)
```

Comment:
error handling logic are the same, can merge with previous line

```python
except (requests.exceptions.HTTPError, requests.exceptions.InvalidJSONError) as details:

```

Reply:
Fixed in https://github.com/app-sre/qontract-reconcile/commit/dbc08b35e413f2de715929237f2f433bd2cfd884